index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
@michael-ciniawsky another thing that I found in the loader, is a way to specify the config location, maybe something like this would be helpful:

```
        var configPath = file.dirname
        if (contextOptions.config) {
            if (path.isAbsolute(contextOptions.config)) {
                configPath = contextOptions.config
            } else {
                configPath = path.join(file.base, contextOptions.config)
            }
        }
```